data/hails.txt

```diff
@@ -2030,7 +2030,288 @@ phrase "friendly author"
 
 phrase "friendly disabled"
 	word
-		"Our ship has been disabled! Please come board our ship and patch us up!"
+		"Our ship has been disabled!"
+		"We've gotten ourselves in a bit of a situation!"
+		"We've lost control of our ship!"
+		"Our ship took quite the pounding!"
+		"Hey!"
+		"S.O.S! S.O.S!"
+	word
+		" "
+	word
+		"Please come board our ship and patch us up!"
+		"We would appreciate any help getting us running again!"
+		"We can't fix this on our own!"
+		"Could you fly by and help us please?"
+		"Requesting emergency repairs!"
+		
+phrase "friendly disabled"
+	word
+		"Hey!"
+		"You're a friendly, correct? "
+		"You answered! "
+		"Good to see a friendly face! "
+		"Hey, please! "
+		""
+		""
+		""
+		""
+		""
+	word
+		"Our ship is disabled,"
+		"We're kinda stuck here,"
+		"We're in a bit of a bind here,"
+		"We're stuck drifting around in this deathtrap,"
+		"Not to be a bother, but"
+		"Could you come over, and"
```

Comment:
this doesn't really work with several combinations
"Could you come over, and [could you lend a hand / ccan you get us back on our feet / would you help us" etc.
basically they ask if you can come over, and they ask for repairs
the rest of the variants of this word seem to work, so removing this one seems to be the easiest way